cfgov/templates/find_a_housing_counselor.html

```diff
@@ -5,6 +5,12 @@
 Find a housing counselor
 {% endblock %}
 
+{% block analytics_js %}
+<!-- Google Optimize --> 
```

Comment:
I'm ok with not calling super as long as we're sure they aren't triggering events in their codebase by referencing `_gaq`.